test/Rings/mpoly-local.jl

```diff
@@ -5,8 +5,8 @@
   I = ideal(Q, [x - 2, (y - 1)^2*z])
```

Comment:
Since the corresponding src file is removed, this file should go as well. The contained tests could get moved to `test/Rings/mpoly-localization.jl` instead